android/app/build.gradle

```diff
@@ -132,8 +129,6 @@ dependencies {
     compile fileTree(dir: "libs", include: ["*.jar"])
     compile "com.android.support:appcompat-v7:23.0.1"
     compile "com.facebook.react:react-native:+"  // From node_modules
-    compile project(':react-native-image-picker')
```

Comment:
FYI, these changes were made when I ran `react-native upgrade`.

Without doing that, I could not launch on my mac.

What version of react-native are you using?